specs/airr-schema.yaml

```diff
@@ -1981,3 +2036,34 @@ CloneTree:
         clone_id:
             type: string
             description: Identifier for the clone.
+        newick_string:
+            type: string
+            description: Newick string of the tree
+        node_list:
+            type: array
+            description: Nodes in the tree
+            items:
+                - $ref: '#/Node'
+                
+# 1-to-n relationship between Tree and Node
+Node:
+    discriminator: AIRR
+    type: object
+    properties:
+        node_id:
+            type: string
+            description: Identifier for this node, matches id in newick string
+        tree_id:
+            type: string
+            description: Identifier for the tree.
+            $ref: '#/Tree'
+        clone_id: #maybe not strictly necessary
+            type: string
+            description: Identifier for the clone.
+            $ref: '#/Clone'
+        sequence:
+            type: string
+            description: DNA sequence of this node
+        meta:
+            type: object #????
```

Comment:
@scharch Is your question whether `type: object` is valid (it is) or just putting question marks to indicate this still being up in the air as far as format for node metadata?

Reply:
@eharkins the question is whether an `object` is the best/most appropriate type. My assumption is a list of arbitrary and undefined properties isn't valid so they have to get put inside an object, but maybe there's a simpler way to handle it.

Reply:
Object makes sense to me for now unless others have suggestions for an improvement.